docs/src/main/sphinx/functions/json.rst

```diff
@@ -2,6 +2,1128 @@
 JSON functions and operators
 ============================
 
+The SQL standard describes functions and operators to process JSON data. They
+allow you to access JSON data according to its structure, generate JSON data,
+and store it persistently in SQL tables.
+
+Importantly, the SQL standard imposes that there be no dedicated data type to
+represent JSON data in SQL. Instead, JSON data shall be represented as
+character or binary strings. Although Trino supports ``JSON`` type, it is not
+used or produced by the following functions.
+
+Trino supports three functions for querying JSON data:
+:ref:`JSON_EXISTS<json_exists>`,
+:ref:`JSON_QUERY<json_query>`, and :ref:`JSON_VALUE<json_value>`. Each of them
+is based on the same mechanism of exploring and processing JSON input using
+JSON path.
+
+JSON path language
+------------------
+
+The JSON path language is a special language, used exclusively by certain SQL
+operators to specify the query to perform on the JSON input. Although the path
```

Comment:
SImplify that sentence. Maybe

The syntax of JSON path expressions significantly differs from SQL. JSON path expressions can be embedded in SQL queries. The semantics of predicates, operators, etc in JSON path expressions generally follow the semantics of SQL. JSON path expressions are case-sensitive for keywords and identifiers.

Reply:
I tried to simplify it.